phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -235,6 +272,11 @@ public static IndexVerifyType fromValue(byte[] value) {
     private static final Option END_TIME_OPTION = new Option("et", "endtime",
             true, "End time for indextool rebuild or verify");
 
+    private static final Option DISABLE_LOGGING_OPTION = new Option("dl",
+        "disable-logging", true
+        , "Disable logging of failed verification rows for BEFORE, " +
```

Comment:
So, we cannot disable logging for ONLY mode?

Reply:
@priyankporwal ONLY counts as BEFORE.